webpack/tests/simple/webpack.config.js

```diff
@@ -1 +1,8 @@
+if (process.env.MY_ENV != 'webpack/tests/simple/index.js') {
```

Comment:
I'm confused. Doesn't an execpath include the three output tree prefixes?

Reply:
This is a source file so it doesn't in this case

Reply:
Ah, right.